packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -151,7 +151,7 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 		ref: mergedRefs,
 		id: `block-${ clientId }${ htmlSuffix }`,
 		tabIndex: 0,
-		role: 'group',
+		role: 'document',
```

Comment:
Was trying to read a bit about this change here https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Document_Role it does seem to be a bit impactful. The link seem to suggest that the role is more suited for elements that should be read and not interacted with and this is not the case for blocks as we should be able to navigate inside blocks and interact with them in different ways depending on the block.

Would you mind sharing more about this change, Is this just for the navigation mode? I wonder if it's something that could have a bad impact on edit mode.

Reply:
@youknowriad No, this is to make older versions of JAWS work in edit mode. From my testing, I can't see that it has hurt in anyway how screen readers interact with the block. However, I flagged this for the Accessibility Team to test that way we can be confident that it has been tested on different systems.